src/utils/pipeline-run-filter-transform.ts

```
@@ -0,0 +1,76 @@
+            : `${k}=${v}` // e.g., "metadata.name=my-pipeline-run"
+        })
+        .join(',')
+    : undefined;
+
+  
+  const matchExpressions: MatchExpression[] = [
+    ...(filterBy.matchExpressions ?? []),
+  ];
+
+  // Build the final selector (excluding custom filter fields that are handled separately)
+  const { filterByName, filterByCreationTimestampAfter, filterByCommit, ...rest } = filterBy;
```

ESLint

[error] 'filterByName' is assigned a value but never used. Allowed unused vars must match /React/u.

(@typescript-eslint/no-unused-vars)